### DISCUSSION TITLE
Capstone ML

### DISCUSSION THREAD

**Discussion by dimasrepo** [teamalgoritma/community#895] [Q&A] [answered]
Dear Team, 
Kendala lubridate.

Setelah di rubah menjadi posixct file, variable start_time harus di bulatkan ke hour menggunakan fungsi round_date  tetapi setelah dicoba berulang kali tidak ada perubahan, hal ini berlaku juga dengan floor_date dan ceilling_date. Mohon bantuannya untuk kendala ini. 

Berikut screenshootnya 
1. chr to date time
![image](https://github.com/user-attachments/assets/f10b6c24-69b4-494b-8dd1-eae44277a0c5)
![image](https://github.com/user-attachments/assets/4a4a1181-9b92-40a2-ad47-286362a8f7ac)

3. date time to round_date 
![image](https://github.com/user-attachments/assets/224f988e-0be6-4475-89b3-2be045668683)

Kenapa ya lubridate sering eror gini mbak/mas ?
Terimakasih atas perhatiannya

Best Regards,

Dimas

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear Pak @dimasrepo 
Terima kasih telah bertanya di Github Community Algoritma. 

Berdasarkan screenshot yang Bapak lampirkan, sepertinya hasil `start_time` yang sudah dibulatkan tersimpan pada kolom baru, pak, di kolom `start_time_rounded`. Kolom baru secara otomatis akan ditambahkan sebagai kolom bagian belakang dari data. Bisa coba dipastikan dulu, pak, apakah kolom `start_time_rounded`-nya sudah sesuai dengan output yang diinginkan? Karena dari sintaks kode sepertinya tidak ada kesalahan. Parameter `"hour"` untuk membulatkan ke jam juga sudah ditambahkan. 

Jika masih belum sesuai, silakan reply pada thread berikut.

Best regards,
Team Algoritma

**Reply by dimasrepo** [reply]
Dear Mbak Saski | Team Algoritma,

Oh iyaa mbak betul setelah di chek ada di akhir yaa kolom round _timenya

Terimakasih atas bantuannya

Best Regards,
Dimas
